Core/Core/Grades/Model/GradeListInteractor.swift

```diff
@@ -193,7 +193,7 @@ public final class GradeListInteractorLive: GradeListInteractor {
                 userID: userID,
                 gradingPeriodID: gradingPeriodID,
                 types: ["StudentEnrollment"],
-                states: [.active]
+                states: [.active, .completed]
```

Comment:
For your information, including `completed` state here at `ReactiveStore` setup is sufficient to solve the issue. 
But to avoid possible issues, I kept checking for enrollment with `completed` state as precaution in these methods:
`gradeEnrollment(from:)`
`course.enrollmentForGrades(userId:includingCompleted)`